rebar.config

```diff
@@ -60,10 +60,10 @@
                    {ref, "9ff46af"}}},
 
         {aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
-                     {ref,"56cf62b"}}},
+                     {ref,"1526ad3"}}},
 
         {aesophia, {git, "https://github.com/aeternity/aesophia.git",
-                     {ref,"faa0ef9"}}},
+                     {ref,"386419f"}}},
```

Comment:
Would it be possible to place comment in this line re how versioning of aesophia relates to this repo?

I see no evident impact to this repo (aeternity/aeternity) though I have the impression that aesophia version shall be bumped for the changes in this commit - maybe for releases from aeternity/aesophia?

Reply:
aesophia should be removed from aeternity node (only kept as a test dependency)

Reply:
Robert has started this work but not finished